loops_lab.js

```diff
@@ -0,0 +1,195 @@
+//1. Write a `while` loop and a `for` loop that takes the variable "num" and logs all the numbers, descending, between "num" and 1.
+
+console.log('1.--------------');
+console.log('While Loop');
+
+let num = 15;
+let startingNum = num;
+while (num > 1 && num) {
```

Comment:
Good use of truthy value here with && num but if statement inside while loop isn't really necessary in this case.